IPython/core/display.py

```diff
@@ -278,6 +280,14 @@ def display(*objs, include=None, exclude=None, metadata=None, transient=None, di
         from IPython.display import display
 
     """
+    # Check whether display() has been called from IPython shell or from Pure Python shell
+    from IPython.core.interactiveshell import InteractiveShell
```

Comment:
We already import `InteractiveShell `  line 303, id you need it here please also remove the import below.